src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,44 @@
 import React from 'react';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
-);
+import './PostInfo.scss';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+
+import { Comment } from '../../types/comment';
+import { User } from '../../types/user';
+import { Post } from '../../types/post';
+
+type Props = {
+  comments: Comment[],
+  users: User[],
+  post: Post,
+};
+
+export const PostInfo: React.FC<Props> = ({ comments, users, post }) => {
+  const commentsForThisPost: Comment[] = comments
+    .filter((comment: Comment) => comment.postId === post.id);
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{post.title}</h3>
+
+        <p>
+          {' Posted by  '}
+
+          <UserInfo user={users.find((u: User) => u.id === post.userId)} />
```

Comment:
Consider handling the case when the user is not found, as users.find() may return undefined. You can provide a default value or display a message.

Reply:
I'm doing these actions in UserInfo block